Library/Formula/glyr.rb

```diff
@@ -0,0 +1,20 @@
+class Glyr < Formula
+  desc "Music related metadata searchengine with command-line interface and C API"
```

Comment:
Sorry, copied it straight from the home page.  The homebrew audit noticed `commandline` but not that one :)